src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2861,9 +2861,9 @@ public override IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predi
 
         #endregion
 
-        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerManager analyzerManager)
+        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerManager analyzerManager, bool reportDiagnosticsWithSourceSuppression)
```

Comment:
minor: It may be a good idea to keep the same name as above (i.e.  includeDiagnosticsWithSourceSuppression) for consistency and for ease of search / replace in the future.